website/addons/base/__init__.py

```diff
@@ -240,6 +239,17 @@ def download_url(self):
 
     @property
     def mfr_download_url(self):
+        url = self._base_butler_url
+        url.path.add('file')
+        url.args['mode'] = 'render'
+
+        if self.revision:
+            url.args[self.version_identifier] = self.revision
+
+        return url.url
+
+    @property
+    def public_download_url(self):
         if settings.DEBUG_MODE:
```

Comment:
check to see if we can remove this offline domain patch with the frontend changes.